macOS/unwindmacos.cpp

```diff
@@ -465,27 +493,47 @@ size_t UnwindGeneratorMacOS::MaybeRedirectExecution(ModuleInfo* module, size_t I
   // now it's the same as fde_array_end, but that might change in the future
   // if other stuff gets written before the next line
   size_t continue_address = tinyinst_.GetCurrentInstrumentedAddress(module);
+#if ARM64
+  if(continue_address & 0x3) {
```

Comment:
Good thinking that on ARM a padding is needed before inserting code. Is this also needed in WriteCustomPersonality?

Reply:
yes!